infrastructure/modules/armonik/ingress-configmap.tf

```diff
@@ -38,11 +44,18 @@ server {
 
     sendfile on;
 
+    if ($accept_language ~ "^$") {
+	set $accept_language "en";
+    }
+
     location = / {
-        rewrite ^ $scheme://$http_host/admin/ permanent;
+        rewrite ^ $scheme://$http_host/admin/$accept_language/ permanent;
```

Comment:
I would say that because `$accept_language` is not constant, the redirection should not be permanent.